src/AppBundle/Entity/Administrator.php

```diff
@@ -44,6 +45,13 @@ class Administrator implements UserInterface
     private $password;
 
     /**
+     * @var string|null
+     *
+     * @ORM\Column(nullable=true)
+     */
+    private $googleAuthenticatorSecret;
```

Comment:
Does it really need to be persisted, or just to be checked once to trust the user?

Maybe saving a date would be more useful, i.e to handle expiration before asking again.